pymesync.py

```diff
@@ -535,6 +535,10 @@ def __local_auth_error(self):
 
     def __response_to_python(self, response):
         """Convert response to native python list of objects"""
+        # DELETE returns an empty body if successful
+        if not response.text and response.status_code == 200:
```

Comment:
@ElijahCaine response.text is None upon successful DELETE requests according to https://github.com/osuosl/timesync/blob/develop/source/draft_api.rst#delete-endpoints . I also checked that this code check works with the staging instance we have up for timesync-node.